src/index.js

```diff
@@ -54,36 +59,44 @@ function determineValidID(event) {
   }
 }
 
+function loadTravelerInfo(userID) {
+  fetchRequests.checkData(userID).then(data => {
+    currentTraveler = new Traveler(data[0])
+    destinationsData = data[1]
+    currentTrips = data[2].filter(trip => trip.userID === currentTraveler.id)
+    currentTraveler.determineTrips(currentTrips, now, destinationsData)
+    displayMainDashboard()
+  })
+}
+
 function displayMainDashboard() {
   domUpdates.changePageDisplay('dashboard')
   domUpdates.greetTraveler(currentTraveler)
   domUpdates.displayCurrentLocation(currentTraveler)
   domUpdates.displayTripCostTotal(currentTraveler, now.split('/')[0])
   domUpdates.displayTrips(currentTraveler.allTrips, tripCardsSection, "My Trips")
 }
+
 function displayUpcomingTripView() {
   domUpdates.changePageDisplay('dashboard')
   domUpdates.displayTrips(currentTraveler.futureTrips, tripCardsSection, "My Upcoming Trips")
 }
+
 function displayPastTripView() {
   domUpdates.changePageDisplay('dashboard')
   domUpdates.displayTrips(currentTraveler.pastTrips, tripCardsSection, "My Past Trips")
 }
+
 function displayPendingTripView() {
   domUpdates.changePageDisplay('dashboard')
   domUpdates.displayTrips(currentTraveler.pendingTrips, tripCardsSection, "My Pending Trips")
 }
-function loadTravelerInfo(userID) {
-  fetchRequests.checkData(userID).then(data => {
-    currentTraveler = new Traveler(data[0])
-    destinationsData = data[1]
-    currentTrips = data[2].filter(trip => trip.userID === currentTraveler.id)
-    currentTraveler.determineTrips(currentTrips, now, destinationsData)
-    displayMainDashboard()
-  })
-}
+
 function exitTripDetailCard(event) {
-  if (event.target === document.getElementById('exitTripDetails')) {
+  if (event.target === document.querySelector('#bookTripButton')) {
```

Comment:
I really like how you've avoided some extra globals for your `querySelectors( )`, as your code looks much cleaner.  This is especially true if they are only used once.

Reply:
I agree with Nicole, however, bookTripButton is already globally declared, so I would use that.

Reply:
I agree with Nicole, however, the bookTripButton is already declared globally, so I would use that instead